pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -309,6 +309,19 @@ func convertToEnvoyJwtConfig(jwtRules []*v1beta1.JWTRule) *envoy_jwt.JwtAuthenti
 	}
 }
 
+func (a *v1beta1PolicyApplier) getMutualTLSModeForPort(endpointPort uint32) model.MutualTLSMode {
+	if a.consolidatedPeerPolicy == nil {
+		return model.MTLSPermissive
+	}
+	if a.consolidatedPeerPolicy.PortLevelMtls != nil {
```

Comment:
What's the expected behavior for passthrough filter chain that doesn't have the port?

Reply:
The behavior should use the workload level mTLS (i.e a.consolidatedPeerPolicy.Mtls). Implementation-wise, you can change this function to treat a special value (i.e `0`) to ignore looking up portLevel settings.